gooddata-sdk/gooddata_sdk/catalog/organization/entity_model/jwk.py

```diff
@@ -0,0 +1,89 @@
+# (C) 2022 GoodData Corporation
+from __future__ import annotations
+
+from typing import List, Optional, Type
+
+import attr
+
+from gooddata_api_client.model.json_api_jwk_in import JsonApiJwkIn
+from gooddata_api_client.model.json_api_jwk_in_attributes import JsonApiJwkInAttributes
+from gooddata_api_client.model.json_api_jwk_in_attributes_content import JsonApiJwkInAttributesContent
+from gooddata_api_client.model.json_api_jwk_in_document import JsonApiJwkInDocument
+from gooddata_sdk.catalog.base import Base
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class CatalogJwkDocument(Base):
+    data: CatalogJwk
+
+    @staticmethod
+    def client_class() -> Type[JsonApiJwkInDocument]:
+        return JsonApiJwkInDocument
+
+    def to_api(self) -> JsonApiJwkInDocument:
+        dictionary = self._get_snake_dict()
+        return self.client_class().from_dict(dictionary, camel_case=False)
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class CatalogJwk(Base):
+    id: str
+    attributes: Optional[CatalogJwkAttributes] = None
+
+    @staticmethod
+    def client_class() -> Type[JsonApiJwkIn]:
+        return JsonApiJwkIn
+
+    @classmethod
+    def init(
+        cls,
+        jwk_id: str,
+        attributes: Optional[CatalogJwkAttributes] = None,
+    ) -> CatalogJwk:
+        return cls(id=jwk_id, attributes=attributes)
```

Comment:
So that you know, the explicit init class method should only be used to simplify users to initialize the class. The same thing you have here in the body of the init method can be done using the default constructor.

`CatalogJwk(id=jwk_id, attributes=attributes)`

I propose to have the following init:

```python
@classmethod
    def init(
        cls,
        id: str,
        rsa_spec: Optional[CatalogRsaSpecification] = None,
    ) -> CatalogJwk:
    return cls(id=id, attributes=CatalogJwkAttributes(content=rsa_spec))
```